src/item-navigator.ts

```diff
@@ -0,0 +1,532 @@
+        position: fixed;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        z-index: 9;
+      }
+
+      .loading-area {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        height: inherit;
+        min-height: inherit;
+      }
+
+      .loading-view {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        min-height: inherit;
+        height: inherit;
+      }
+
+      ia-itemnav-loader {
+        display: block;
+        width: 100%;
+      }
+
+      .hidden {
+        height: 1px;
+        width: 1px;
+      }
+
+      button {
+        cursor: pointer;
+        padding: 0;
+        border: 0;
+      }
+
+      button:focus,
+      button:active {
+        outline: none;
+      }
+
+      .menu-and-reader {
+        position: relative;
+      }
+
+      nav button {
+        background: none;
+      }
+
+      nav .minimized {
+        background: rgba(0, 0, 0, 0.7);
+        padding-top: 6px;
+        position: absolute;
+        width: ${menuMargin};
+        z-index: 2;
+        left: 0;
+        border-bottom-right-radius: 5%;
+      }
+
+      nav .minimized button {
+        width: var(--iconWidth);
+        height: var(--iconHeight);
+        margin-bottom: 0.2rem;
+        margin: auto;
+        display: inline-flex;
+        vertical-align: middle;
+        -webkit-box-align: center;
+        align-items: center;
+        -webkit-box-pack: center;
+        justify-content: center;
+        width: ${menuMargin};
+        height: ${menuMargin};
+      }
+
+      nav .minimized button.toggle-menu > * {
+        border: 2px solid var(--iconStrokeColor);
+        border-radius: var(--iconWidth);
+        width: var(--iconWidth);
+        height: var(--iconHeight);
+        margin: auto;
+      }
+
+      ia-icon-ellipses {
+        width: var(--iconWidth);
+        height: var(--iconHeight);
+      }
+
+      #menu {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        z-index: 3;
+        overflow: hidden;
+        transform: translateX(-${subnavWidth});
+        width: ${subnavWidth};
+        transform: translateX(calc(${subnavWidth} * -1));
+        transition: ${transitionEffect};
+      }
+
+      #reader {
+        position: relative;
+        z-index: 1;
+        /* transition: ${transitionEffect}; */
+        transform: translateX(0);
+        width: 100%;
+      }
+
+      .open.overlay #reader {
+        transition: none;
+      }
+
+      .open #menu {
+        width: ${subnavWidth};
+        transform: translateX(0);
+        transition: ${transitionEffect};
+      }
+
+      .open.shift slot[name='theater-header'],
+      .open.shift #reader {
+        width: calc(100% - var(--menuWidth));
+        float: right;
+        transition: ${transitionEffect};
+      }
+    `;
+  }
+}
+
+customElements.define('ia-menu-slider', IAMenuSlider);
```

Comment:
Do you still need this?

Reply:
thanks for catching, just updated package.